cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/Extensions.kt

```diff
@@ -1067,13 +1067,15 @@ val Node?.assigns: List<AssignExpression>
  *
  * @param predicate the search predicate
  */
-fun Node.firstParentOrNull(predicate: (Node) -> Boolean): Node? {
+inline fun <reified T : Node> Node.firstParentOrNull(
```

Comment:
Two things
1) The description should be adjusted to reflect that this selects the first parent of type [T] that matches the (optional) predicate
2) It would be nice to also have this for the firstParentScope function.

Reply:
LGTM, minor documentation issue and possibly a small enhancement